ci/infra/azure/variables.tf

```diff
@@ -0,0 +1,175 @@
+variable "azure_location" {
+  type = string
+  # default     = "West Europe"
+  description = "Name of the AZURE location to be used"
+}
+
+variable "enable_zone" {
+  type        = bool
+  default     = false
+  description = "Use this if only zone is available in the deploying region"
+}
+
+variable "azure_availability_zones" {
+  type        = list(string)
+  default     = ["1", "2", "3"]
+  description = "List of Availability Zones (e.g. `['1', '2', '3']`)"
```

Comment:
By default the AZ are disabled, I would prefer to drop the `default` and update the description to include some real values for the example Azure Location we mentioned inside of `azure_location`

Reply:
changed